test/test_extractor.py

```diff
@@ -167,7 +167,15 @@ class TestExtractFileZip(TestExtractorBase):
         when the exe is a self-extracting zipfile """
 
     def setup_method(self):
-        for filename in ["test.exe", "test.zip", "test.jar", "test.apk", "test.msi"]:
+        for filename in [
+            "test.exe",
+            "test.zip",
+            "test.jar",
+            "test.apk",
+            "test.msi",
+            "test.egg",
+            "test.whl",
```

Comment:
@terriko, I have already added tests for this.

Reply:
I should have been more precise; I was thinking about long tests with an actual file or two. pyca-cryptography should give us a few actual hits on openssl and would be a good option I think?

Reply:
I have tested it locally and it is showing openssl and glibc. I have also added it to longtest of openssl.
```console
           WARNING  cve_bin_tool.VersionScanner - python was detected with version UNKNOWN in file                          version_scanner.py:142
                    C:\Users\Root\AppData\Local\Temp\cve-bin-tool-d81xxr55\cryptography-3.0-cp35-abi3-manylinux2010_x86_64…
[01:30:23] INFO     cve_bin_tool -                                                                                                      cli.py:274
           INFO     cve_bin_tool - Overall CVE summary:                                                                                 cli.py:275
           INFO     cve_bin_tool - There are 3 products with known CVEs detected                                                        cli.py:277
           INFO     cve_bin_tool - Known CVEs in ('openssl', '1.1.1.6'), ('glibc', '2.2'):                                              cli.py:286

```

Reply:
looks good, thanks!